Tasks/ACRTaskV0/task.json

```diff
@@ -0,0 +1,209 @@
+{
+    "id": "769D88CB-515B-4456-A045-D9A4E11C90E3",
+    "name": "ACRTaskV0",
+    "friendlyName": "ACR Task",
+    "description": "Build and run container images using Azure Container Registry (ACR) tasks",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=851275) or [see the ACR Task documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-tasks-overview)",
+    "category": "Build",
+    "visibility": [
+        "Build",
```

Comment:
mark it as preview task. Add preview flag.

Reply:
done